app/Services/DiscordService.php

```diff
@@ -158,6 +159,13 @@ public function refresh()
         ];
         $content = $this->post('oauth2/token', $body);
         $this->saveUserApp($content);
+        $log = 'Renewed user #' . $this->user->id . ' Discord auth token.';
+        $this->logs[] = $log;
+
+        JobLog::create([
```

Comment:
This is going to create one JobLog when the user renews themselves (this not on a job), and also create a log for every user (potentially 100s a day), please refactor to only do a single log and just for the command :) Also only needs to create the log if the local install has jog logs (not available for self-hosted kanka)